src/Temporalio/Workflows/WorkflowQueryAttribute.cs

```diff
@@ -37,6 +37,12 @@ public WorkflowQueryAttribute(string name)
         /// </summary>
         public string? Name { get; }
 
+        /// <summary>
+        /// Gets or sets a short description for this query that may appear in UI/CLI.
```

Comment:
Maybe worth mentioning that this is returned when asking what queries a workflow supports, and that _that_ is what may appear in UI/CLI